backend/src/bundles/users/types/user-update-response-dto.ts

```diff
@@ -0,0 +1,14 @@
+import { type Gender } from './types.js';
+
+type UserUpdateResponseDto = Partial<{
```

Comment:
Create normal entity, without Partial

Reply:
fix in [https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/98/commits/62a76c664ba69ef6318c1757a3e2d6e18969cec6](url)